include/admin_navigation.js

```diff
@@ -56,115 +58,150 @@ module.exports = function AdminNavigationModule(pb) {
-                    access: SecurityService.ACCESS_EDITOR
-                },
-                {
-                    id: 'pages',
-                    title: 'PAGES',
-                    icon: 'file-o',
-                    href: '/admin/content/pages',
-                    access: SecurityService.ACCESS_EDITOR
-                },
-                {
-                    id: 'articles',
-                    title: 'ARTICLES',
-                    icon: 'files-o',
-                    href: '/admin/content/articles',
-                    access: SecurityService.ACCESS_WRITER
-                },
-                {
-                    id: 'media',
-                    title: 'MEDIA',
-                    icon: 'camera',
-                    href: '/admin/content/media',
-                    access: SecurityService.ACCESS_WRITER
-                },
-                {
-                    id: 'comments',
-                    title: 'COMMENTS',
-                    icon: 'comments',
-                    href: '/admin/content/comments',
-                    access: SecurityService.ACCESS_EDITOR
-                },
-                {
-                    id: 'custom_objects',
-                    title: 'CUSTOM_OBJECTS',
-                    icon: 'sitemap',
-                    href: '/admin/content/objects/types',
-                    access: SecurityService.ACCESS_EDITOR
-                }
-            ]
-        },
-        {
-            id: 'plugins',
-            title: 'PLUGINS',
-            icon: 'puzzle-piece',
-            href: '#',
-            access: SecurityService.ACCESS_ADMINISTRATOR,
-            children: [
-                {
-                    divider: true,
-                    id: 'manage',
-                    title: 'MANAGE',
-                    icon: 'upload',
-                    href: '/admin/plugins'
-                },
-                {
-                    id: 'themes',
-                    title: 'THEMES',
-                    icon: 'magic',
-                    href: '/admin/themes'
-                }
-            ]
-        },
-        {
-            id: 'users',
-            title: 'USERS',
-            icon: 'users',
-            href: '#',
-            access: SecurityService.ACCESS_EDITOR,
-            children: [
-                {
-                    id: 'manage',
-                    title: 'MANAGE',
-                    icon: 'users',
-                    href: '/admin/users',
-                    access: SecurityService.ACCESS_EDITOR
-                },
-                {
-                    id: 'permissions',
-                    title: 'PERMISSIONS',
-                    icon: 'lock',
-                    href: '/admin/users/permissions',
-                    access: SecurityService.ACCESS_ADMINISTRATOR
-                },
-            ]
-        },
-        {
+    var MULTISITE_NAV = Object.freeze({
+            id: 'site_entity',
+            title: 'MANAGE_SITES',
+            icon: 'sitemap',
```

Comment:
@blakecallens do we need to add this or are y'all taking care of this on your side?